release.json

```diff
@@ -41,24 +22,9 @@
         "WINDOWS_APMINJECT_MODULE": "release-signed",
         "WINDOWS_APMINJECT_VERSION": "1.1.3",
         "WINDOWS_APMINJECT_SHASUM": "5fdd62a84e640204386b9c28dc2e3ac5d9b8adde6427cb9f5914619f94d7b5bd",
-        "INTEGRATIONS_CORE_VERSION": "25613230b069bf89f1cc63b8eaf2aa54be42ed91"
-    },
-    "release-a6": {
-        "INTEGRATIONS_CORE_VERSION": "7.56.0-rc.2",
-        "OMNIBUS_SOFTWARE_VERSION": "7.56.0-rc.3",
-        "OMNIBUS_RUBY_VERSION": "7.56.0-rc.1",
-        "JMXFETCH_VERSION": "0.49.6",
-        "JMXFETCH_HASH": "f06bdac1f8ec41daf9b9839ac883f1865a068b04810ea82197b8a6afb9369cb9",
-        "SECURITY_AGENT_POLICIES_VERSION": "v0.64.1",
-        "MACOS_BUILD_VERSION": "6.56.0-rc.3",
-        "WINDOWS_DDNPM_DRIVER": "release-signed",
-        "WINDOWS_DDNPM_VERSION": "2.7.1",
-        "WINDOWS_DDNPM_SHASUM": "0f4665761324e1fef1c21651be5b70e79c72b5e7e5662d74619e7db2b27d5bc2",
-        "WINDOWS_DDPROCMON_DRIVER": "release-signed",
-        "WINDOWS_DDPROCMON_VERSION": "1.0.4",
-        "WINDOWS_DDPROCMON_SHASUM": "3a23804adc7280390aabc01f0b709853755baa111f821f99627cd661ee917490"
+        "INTEGRATIONS_CORE_VERSION": "5e36ab81f77809f25c51c22da049b8bbfca32c61"
```

Comment:
This is moving the Integrations Core version to a commit a few days earlier, is that intentional?